Makefile

```diff
@@ -60,7 +61,7 @@ run: server
 	./$(APP_NAME)
 
 test: 
-	godep go test ./...
+	godep go test $(GOFILES)
```

Comment:
Actually that was the problem before.  This only runs the test in Heketi.  Makes it simple to test all of your code before submitting.